src/components/TaxAssistant.jsx

```
@@ -0,0 +1,99 @@
+                    <div key={index} className={`flex items-start gap-3 ${message.role === 'user' ? 'justify-end' : ''}`}>
+                        {message.role === 'model' && (
+                            <div className="w-8 h-8 rounded-full bg-emerald-100 flex items-center justify-center flex-shrink-0">
+                                🤖
+                            </div>
+                        )}
+                        <div className={`p-3 rounded-lg max-w-lg ${
+                            message.role === 'user'
+                                ? 'bg-blue-600 text-white'
+                                : 'bg-gray-100 text-gray-800'
+                        }`}>
+                            <div className="prose prose-sm" dangerouslySetInnerHTML={{ __html: message.parts[0].text.replace(/\n/g, '<br />') }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)